lib/shims/eq/R2015b/contains.m

```diff
@@ -0,0 +1,9 @@
+function result = contains(str, pat)
+
+% CONTAINS Determine if pattern is in strings
+% CONTAINS Introduced in R2016b
+% https://www.mathworks.com/help/matlab/ref/contains.html
+
+result = length(strfind(str, pat)) > 0;
+
+end  % function contains
```

Comment:
No trailing `% what this "end" corresponds to` comments, please.

Reply:
ok, I will change to honor this style preference, and get this into the open PR